lib/lotus/repository.rb

```diff
@@ -661,6 +662,25 @@ def exclude(query)
         query.negate!
         query
       end
+
+      # This is a method to check entity persited or not
+      # 
+      # @param entity
+      #
+      # @return true
+
+      def persisted?(entity)
+        !!entity.id
+      end
+
+      def update_timestamps(entity)
+        if entity.class.attributes.include?(:created_at)
+          if entity.created_at.nil?
```

Comment:
Sorry what I meant is to remove the check for nil and have this line: `entity.created_at ||= Time.now.utc`